### DISCUSSION TITLE
slice built-in function example

### DISCUSSION THREAD

**Discussion by pbelecky** [google/xls#525] [General]
Hello!
I found a slice built-in function in DSLX, but I don't understand how to use it.
https://google.github.io/xls/dslx_reference/#slice

Please give me an example.
For example, I have 
```f: s32[64]```
how get slice from 5 to 10 element?

**Comment by aminiok1** [comment]
Hi!

I believe this should work:
```
slice(f, u32:5, s32[5]: [s32:0, ...])
```
The second argument specifies the start index (5) and the third argument specifies the desired return type (s32) and the number of elements to slice (5), I have initialized it to 0 but it doesn't matter as it is only used for type deduction.

**Reply by pbelecky** [reply]
Thanks a lot!
